common/compiler/vyper_test.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2015 The go-ethereum Authors
```

Comment:
Same as above, I ran out of idea for goofy comments.

Reply:
😭😭